server/lib/analytics.js

```diff
@@ -8,7 +8,15 @@ const segmentArticle = require('../lib/article/segment');
 const DEBUG = false;
 const BARRIERTYPE = 'trial';
 
-const fixBooleans = data => data.replace('"AUTHDATA(data.access)"', 'AUTHDATA(data.access)');
+const fixAuthData = data => {
+	const authDataString = '"AUTHDATA(access)"';
+
+	if(authDataString === '""') {
```

Comment:
Never true? Or if there's magic happening, good to comment on that fact.

Reply:
When `access` is `false`, amp-analytics for some reason outputs an empty string... Will add a comment explaining this.